src/__tests__/reducer.test.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
```

Comment:
- [ OPTIONAL] Kindly remove the eslint disable line and instead fix the issue in the code.